docs/navigate.rst

```diff
@@ -198,6 +199,13 @@ View Node Dashboard
    c. **APIC Endpoint Details** - The Visibility dashboard displays APIC details for the Node: - 1. MAC 2. EPG 3. Node 4. Interface 5. VLAN Encap 
                                         
    d. **BIG-IP Endpoint Details** - The Visibility dashboard displays BIG-IP details for the Node: - 1. MAC 2. VLAN 3. Interfaces (There is a **View Logs** link beside interfaces to view the interface logs) 4. Self IPs
+   
+6. F5 ACI ServiceCenter supports VXLAN tunnel interfaces along with individual, Port channel, Virtual port channel interfaces on Node table visibility dashboard.
+   
+   To view details of the VXLAN tunnel interfaces, Click on Node IP in Node information table -> Scroll down to APIC Endpoint details table. All interfaces associated with the node like VPC/PC/Individual/tunnel are displayed in the interface column and On click over the interface additional information related to VPC/PC/Individual/tunnel interface will be displayed in the side panel.
```

Comment:
... tunnel interfaces, click **Node IP** in the Node information table, and then scroll down to the APIC Endpoint details table. 

All interfaces associated with the node, such as VPC, PC, Individual, and Tunnel, are displayed in the interface column. When you click the interface, additional information related to these interfaces displays in the side panel.
(that's an attempt to fix that sentence, I really wasn't sure what '.. and On click over the interface..' meant.)